lib/datasource/local/settings_storage.dart

```diff
@@ -25,6 +25,7 @@ const String _kTokensWhiteList = 'tokens_whitelist';
 const String _kIsCitizen = 'is_citizen';
 const String _kIsFirstRun = 'is_first_run';
 const String _kIsFirstTimeOnDelegateScreen = 'is_first_time_on_delegate_screen';
+const String _kDateSinceLastAsked = 'date_since_last_asked';
```

Comment:
I think `date_since_last_asked` in not clear enough how about `date_since_last_rate_asked`?

Reply:
even better

`time_since_rate_app_promted `

Reply:
`since` would indicate a duration between now and the saved date, not the last time it happened 🤓 Or are we trying to indicate that it's `millisecondsSinceEpoch`? How about `last_date_prompted_for_inapp_review`? I feel `date` sufficiently indicates millisecondsSinceEpoch 🤓

Reply:
all better names, I changed to kDateSinceRateAppPrompted = 'date_since_rate_app_prompted' . Thanks